peer/peerlist/list.go

```diff
@@ -568,3 +584,24 @@ func (pl *List) Introspect() introspection.ChooserStatus {
 		Peers: peersStatus,
 	}
 }
+
+// shuffle randomizes the order of a slice of peers.
```

Comment:
May as well comment that this is Fisher-Yates.

Reply:
done.